lib/jekyll/converters/smartypants.rb

```diff
@@ -3,9 +3,15 @@
 class Kramdown::Parser::SmartyPants < Kramdown::Parser::Kramdown
   def initialize(source, options)
     super
-    @block_parsers = [:block_html]
+    @block_parsers = [:block_html, :content]
     @span_parsers =  [:smart_quotes, :html_entity, :typographic_syms, :span_html]
   end
+
+  CONTENT_MATCH = %r!^.*\n!
```

Comment:
It'd be better to have the constant moved to the top of the class (before `:initialize`) like how it is for classes under Jekyll namespace..

Reply:
Good call :+1:

Reply:
Would it be better to just inline it?
I wanted to keep it close to where it is being used. If this class gets too complicated, it should be split out into a separate file. Maybe it's time for that anyway?

Reply:
If it has to be a regex, then lets inline it with a frozen definition.. (so its still a constant..)
Regarding the move to a new file.., nope.. I'm against it because, this is not a class in our namespace..

Reply:
It would live at `lib/kramdown/parser/smartypants.rb`

Reply:
Prior discussion: https://github.com/jekyll/jekyll/pull/4323#issuecomment-169668222

Reply:
true.. but if core is going to consider making the switch to `CommonMark`, I feel this abstraction is futile..